python-client/giskard/integrations/mlflow/giskard_evaluator_utils.py

```diff
@@ -0,0 +1,126 @@
+import inspect
+import re
+import logging
+
+import langchain.chains.base
+
```

Comment:
This import will break if langchain is not installed. We usually do:
```python
try:
    import giskard_llm_dependency
except ImportError as err:
    raise LLMImportError() from err
```
except in cases where the library is implicitly available (e.g. a user instantiating a `LangchainModel`). I don’t know if this is the case, could you check?